docs/UserGuide.md

```diff
@@ -63,6 +67,36 @@ Click on the relevant links to easily navigate through the guide and access the
 
 1. Refer to the [Features](#features) below for details of each command.
 
+--------------------------------------------------------------------------------------------------------------------
+## UI Components
+
+![generalUi](images/generalUi.png)
+
+### General UI information
+
+Component | Purpose
+--------|------------------
+**Navigation Bar** | Allows you to exit UNOFAS or view help
+**Command Line** | Location to enter commands
+**Result Line** | Displays the result after a command is entered
+**Contact List** | Displays clients
+**Appointment List** | Displays appointments
+**Contact Card** | Displays detailed information about a patient
+**Appointment Card** | Displays detailed information about an appointment
+**Save Location** | Displays the location where your UNOFAS data is stored
+
+### Contact Card
+
+![contactCardUi](images/contactCardUi.png)
+
+* Shows the client's name, phone number, financial plan(s) if any, address, email address, next-of-kin name, next-of-kin phone number, appointment (if any) and tags (if any).
```

Comment:
Perhaps it would be better to standardise the `if any` and multiplicity of tags/financial plans here.
```suggestion
* Shows the client's name, phone number, financial plan(s) (if any), address, email address, next-of-kin name, next-of-kin phone number, appointment (if any) and tag(s) (if any).
```